src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -99,214 +113,343 @@ const GET_PROFILE_HEADING_QUERY = {
+            <EditIcon className="editIcon" />
+            <div className="editResetDiv">
+              <ResponsiveButton
+                component={Link}
+                to={`/form/${userInfo.accountType.role}/${userInfo.user.id}`}
+                className="edit"
+              >
+                Edit Profile
+              </ResponsiveButton>
+              {isAdmin &&
+                (userInfo.accountType.role === "student" ? (
+                  <ResponsiveButton
+                    className="resetStudent"
+                    disabled
+                    onClick={handleClickOpen}
+                  >
+                    Reset Password
+                  </ResponsiveButton>
+                ) : (
+                  <ResponsiveButton className="reset" onClick={handleClickOpen}>
+                    Reset Password
+                  </ResponsiveButton>
+                ))}
+            </div>
+            <ResetDialog />
+          </Grid>
+          <Grid component={Hidden} item lgUp xs={12}>
+            <Button
+              component={Link}
+              to={`/form/${userInfo.accountType.role}/${userInfo.user.id}`}
+              variant="outlined"
+            >
+              <EditIcon />
+            </Button>
+          </Grid>
+        </>
+      )}
+    </Grid>
+  );
+
+  const ResetDialog = () => (
+    <>
+      <Dialog
+        open={open}
+        onClose={handleClosePassword}
+        maxWidth="xs"
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title" className="center dialog-padding">
+          {"Do you want to reset this user's password?"}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description" className="center">
+            When you reset the password, a randomly generated password will
+            become the user's new password. This action cannot be reverted. An
+            automatic email will be sent out notifying the user that their
+            password has been reset. The user can also reset their own password
+            through the portal login page.
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions className="dialog-actions">
+          <Button
+            onClick={handleClosePassword}
+            variant="outlined"
+            className="cancelBtn"
+          >
+            cancel
+          </Button>
+          <Button
+            onClick={handleClosePasswordReset}
+            className="resetBtn"
+            autoFocus
+          >
+            reset password
+          </Button>
+        </DialogActions>
+      </Dialog>
+      <Dialog
+        open={openReset}
+        onClose={handleCloseReset}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title" className="center">
+          {"Password has been reset."}
```

Comment:
similar feedback - remove curly brackets + add "disableTypography"

Reply:
Done thank you!